templates/embeds/collapsible.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends '@Tabler/embeds/card.html.twig' %}
+{% set collapsed = false %}
+{% set fullsize = true %}
+{% block box_title %}{% endblock %}
+{% block box_body %}
+    {% set _id = id ?? tabler_unique_id('collapsible_') %}
+    {% set _open = open ?? false %}
+    {% set _id_collapse = _id ~ '_item' %}
+
+    <div id="{{ _id }}" class="accordion accordion-flush{% block class %} {% endblock %}">
+        <div class="accordion-item {% block item_class %} {% endblock %}">
+            <div class="accordion-header" id="{{ _id }}_heading">
+                <button class="accordion-button {% if not _open %}collapsed{% endif %}"
+                        type="button" data-bs-toggle="collapse" data-bs-target="#{{ _id_collapse }}"
+                        aria-expanded="true">
+                    {% block title %}{% endblock %}
+                </button>
+            </div>
+
+            <div id="{{ _id_collapse }}" class="accordion-collapse collapse {% if _open %}show{% endif %}">
+                <div class="accordion-body {% block body_class %}pt-0{% endblock %}">
+                    {% block body %}{% endblock %}
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
I understand that the `accordion` component is scary and not easy to manipulate/use.
But isn't it more "Maintainers" friendly to use it?
The less duplicate code we have to maintain, the happier we are?